docs/Guides/Migration-Guide-V4.md

```diff
@@ -71,28 +71,48 @@ The route registration has been made synchronous from v4.
 This change was done to provide better error reporting for route definition.
 As a result if you specify an `onRoute` hook in a plugin you should either:
 * wrap your routes in a plugin (recommended)
+
+  For example refactor your server to this:
+
+  ```
```

Comment:
For consistency,

For example refactor this:

```js
fastify.register((instance, opts, done) => {
  instance.addHook('onRoute', (routeOptions) => {
    const { path, method } = routeOptions;
    console.log({ path, method });
    done();
  });
});

fastify.get('/', () => 'hello')
```

Into this:

```js
fastify.register((instance, opts, done) => {
  instance.addHook('onRoute', (routeOptions) => {
    const { path, method } = routeOptions;
    console.log({ path, method });
    done();
  });
});

fastify.register((instance, opts, done) => {
  instance.get('/', () => 'hello');
  done();
});
```

Reply:
Done!